tests/tracking/test_rest_tracking.py

```diff
@@ -673,6 +673,31 @@ def assert_bad_request(payload, expected_error_message):
             f"Invalid value foo for parameter '{request_parameter}' supplied",
         )
 
+    ## Should 400 if missing timestamp
+    assert_bad_request(
+        {
+            "run_id": run_id,
+            "metrics": [
+                {"key": "mae", "value": 2.5}
+            ]
+        },
+        'Invalid value [{"key": "mae", "value": 2.5}] for parameter \'metrics\' supplied'
```

Comment:
not that familiar with the test suite -- is there a way to remove the list of values and replace it with a `*` or something to be agnostic to the actual `metrics` list in the request?